### PR TITLE
add shellac

### PR DESCRIPTION
**Pull request by jonborchardt**

### CODE REVIEW

File: packages/varnish/package.json [hunk 1]
```diff
@@ -56,7 +57,7 @@
     "@types/react-transition-group": "^4.4.5",
     "clsx": "^1.2.1",
     "csstype": "^3.1.1",
-    "hex-rgb": "^5.0.0",
+    "hex-rgb": "^4.3.0",
```
Comment (jonborchardt) [inline packages/varnish/package.json:60]: newer hex does not work

File: docs/data/material/varnish/shellac/shellac.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+You should use Shellac for building simple, static sites. If your application is likely to be complex and include a lot of interactivity, use Varnish proper instead.
+
+## Usage
+
+Add this [css](https://cdn.jsdelivr.net/npm/@allenai/varnish2@latest/shellac.css) to your html page.
+
+## What you get
+
+All Varnish2/MUI styles expressed as CSS Variables.
```
Comment (arnavic) [inline docs/data/material/varnish/shellac/shellac.md:18]: is there somewhere that people can go to see the full list of CSS vars? can that be linked or added here?

File: docs/data/material/varnish/shellac/shellac.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: shellac
+githubLabel: 'varnish: shellac'
+---
+
+# Shellac
+
+<p class="description">`Shellac` is a pared down version of `Varnish` that lets you make a polished, AI2 branded website with HTML alone.</p>
```
[the remark this answers is not in the document]
Comment (jonborchardt) [reply docs/data/material/varnish/shellac/shellac.md]: done

File: docs/data/material/varnish/shellac/shellac.md (hunk 2 above)
Comment (jonborchardt) [reply docs/data/material/varnish/shellac/shellac.md:18]: done

**Pull request merged**